src/app/application.cpp

```diff
@@ -292,7 +292,8 @@ Application::Application(int &argc, char **argv)
     connect(this, &QGuiApplication::commitDataRequest, this, &Application::shutdownCleanup, Qt::DirectConnection);
 #endif
 
-    LogMsg(tr("qBittorrent %1 started", "qBittorrent v3.2.0alpha started").arg(QStringLiteral(QBT_VERSION)));
+    LogMsg(tr("qBittorrent %1 started. Process ID: %2", "qBittorrent v3.2.0alpha started")
```

Comment:
I wonder what benefit you think could be gained by seeing the process ID when analyzing logs?

Reply:
It should help differentiate the log message at line 244. Otherwise user would have a hard time to know which process exited and which stayed.

Reply:
Hmm... Seems this log message is being added too early. I wouldn't expect it to be added before qBittorrent is daemonized. I.e. when qBittorrent is launched as a service there should be only one message "qBittorrent is started".

Reply:
I think it is an unwanted side-effect of `app.reset()`. Ideally qbt should try to know whether it should become a daemon and do it as soon as possible (before Application is initialized and therefore we don't need `app.reset()`).
Unfortunately I don't think this is solvable in this PR. 

https://en.wikipedia.org/wiki/Daemon_(computing)#Unix-like_systems